### PR TITLE
👌 IMPROVE: Grid Layout and Merged WebGL Code

### PR DESCRIPTION
**Pull request by ObaidNadeem**
- Grid layout for homepage's content
- Merged the Webgl code

### CODE REVIEW

**Comment by ahmadawais** [comment]
@ObaidNadeem please resolve conflicts.

**Comment by ahmadawais** [comment]
@ObaidNadeem lots of irrelevant files in your PR.

**Comment by ObaidNadeem** [comment]
What kinda ? I only rebased from main

**Comment by ahmadawais** [comment]
@ObaidNadeem please rebase. @msaaddev p1 merge.

File: apps/baseai.dev/src/components/home/hero.tsx [hunk 1]
```diff
@@ -120,6 +119,64 @@ interface CopyableCommandProps {
 	command: string;
 }
 
+function Socials() {
+	return (
+		<>
+			<Link
+				href={'https://github.com/LangbaseInc/'}
```
Comment (msaaddev) [inline apps/baseai.dev/src/components/home/hero.tsx:126]: @ObaidNadeem the link is wrong. It should point to BaseAI repo.

**Review by msaaddev — APPROVED** [review]
LGTM.

**Comment by ahmadawais** [comment]
@ObaidNadeem @msaaddev this has decreased the space below baseai and there's empty space at the top. it can be improved a lot 
image

**Pull request merged**